src/@core/domain/models/city.ts

```diff
@@ -11,6 +11,11 @@ export type City = {
   overallScore: number
   provinceId: string
   short: string
+  industries: string[]
+  nature: boolean
+  population: number
+  festivals: string
```

Comment:
pq 'festivals' eh uma 'string'? Nao seria um 'boolean' ou uma lista de festivals 'string[]'?

Reply:
pensei em ter uma string desse tipi aqui:

"Jazz Festival, Just for Laughs Comedy Festival, and many others"